components/uikit/Buttons/BackIconButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button, ButtonIcon } from "@gluestack-ui/themed";
+import React from "react";
+
+export type BackIconButtonProps = {
+    icon: React.ElementType,
+    isDisabled: boolean,
+    bgColor?: string,
+    pressedBgColor?: string,
+    iconColor?: string,
+    onPress?: () => void;
+}
+
+export const BackIconButton = ({
+  icon,
+  isDisabled,
+  bgColor,
+  pressedBgColor,
+  iconColor,
```

Comment:
I suggest you to add default values for these props

Reply:
Done @BOCOVO . 
I also add a demo (web). Mobile is still not working